src/main/java/AutoLotto/view/ResultView.java

```diff
@@ -61,7 +64,30 @@ private void outputMatchingLottos(Map<PrizeEnum, Integer> lottoResult) {
     }
 
     private String getPrizeResult(PrizeEnum prize, int value) {
-        return String.format("%d%s (%d) - %d개", prize.getMatchedCount(),"개 일치", prize.getPrize(), value);
+        if (isMissMatched(prize)) {
```

Comment:
enum은 직접 비교하는게 더 직관적인 것 같아요
```suggestion
        if (prize == PrizeEnum.MISS) {
```